dnfdaemon-server/transaction.hpp

```diff
@@ -20,15 +20,17 @@ along with dnfdaemon-server.  If not, see <https://www.gnu.org/licenses/>.
 #ifndef DNFDAEMON_SERVER_SERVICES_RPM_TRANSACTION_HPP
 #define DNFDAEMON_SERVER_SERVICES_RPM_TRANSACTION_HPP
 
-#include <libdnf/rpm/transaction.hpp>
 #include <libdnf/rpm/package.hpp>
+#include <libdnf/rpm/transaction.hpp>
 #include <sdbus-c++/sdbus-c++.h>
 
 #include <string>
 
+namespace dnfdaemon {
+
 class RpmTransactionItem : public libdnf::rpm::TransactionItem {
 public:
-    enum class Actions { INSTALL, ERASE, UPGRADE, DOWNGRADE, REINSTALL };
+    enum class Actions { INSTALL, ERASE, UPGRADE, DOWNGRADE, REINSTALL, CLEANUP };
```

Comment:
I just notice `ERASE` - we rather use `REMOVE` in libdnf.
Please check also include/libdnf/transaction/transaction_item_action.hpp if it has any use for you.

Reply:
Well, you are right and unifying these actions would be nice (we currently have three copies - one in libdnf/transaction and two others in microdnf and dnfdaemon). I consider this part of daemon kind of workaround - the code should be removed once we have better higher level transaction API in libdnf.